drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/editor/ScenarioSimulationEditorKogitoWrapper.java

```diff
@@ -362,32 +383,45 @@ protected void onBackgroundTabSelected() {
         scenarioSimulationEditorPresenter.onBackgroundTabSelected();
     }
 
-    protected void newFile(Path path) {
-        Command createCommand = () -> {
-            final ScenarioSimulationModel.Type selectedType = scenarioKogitoCreationPopupPresenter.getSelectedType();
+    /**
+     * It shows the <code>ScenarioSimulationCreationPopup</code> given the file location, used
+     * @param path
+     */
+    protected void showScenarioSimulationCreationPopup(Path path) {
+        scenarioSimulationCreationPopupPresenter.show(ScenarioSimulationEditorConstants.INSTANCE.addScenarioSimulation(),
+                                                      createNewFileCommand(path));
+    }
+
+    /**
+     * It creates the command launched when a user wants to create a new Scenario Simulation file.
+     * @param path
```

Comment:
Could you please add more details on path and return - otherwise remove it?

Reply:
@dupliaka done!